packages/react/src/hooks/useBlockNote.ts

```diff
@@ -2,106 +2,25 @@ import {
   BlockNoteEditor,
   BlockNoteEditorOptions,
   BlockSchema,
+  defaultBlockSchema,
   DefaultBlockSchema,
 } from "@blocknote/core";
-import { DependencyList, FC, useEffect, useState } from "react";
-import { getBlockNoteTheme } from "../BlockNoteTheme";
-import { createReactBlockSideMenuFactory } from "../BlockSideMenu/BlockSideMenuFactory";
-import { DragHandleMenuProps } from "../BlockSideMenu/components/DragHandleMenu";
-import { createReactFormattingToolbarFactory } from "../FormattingToolbar/FormattingToolbarFactory";
-import { createReactHyperlinkToolbarFactory } from "../HyperlinkToolbar/HyperlinkToolbarFactory";
-import { defaultReactSlashMenuItems } from "../SlashMenu/defaultReactSlashMenuItems";
-import { createReactSlashMenuFactory } from "../SlashMenu/SlashMenuFactory";
-
-//based on https://github.com/ueberdosis/tiptap/blob/main/packages/react/src/useEditor.ts
-
-type CustomElements<BSchema extends BlockSchema> = Partial<{
-  formattingToolbar: FC<{ editor: BlockNoteEditor<BSchema> }>;
-  dragHandleMenu: FC<DragHandleMenuProps<BSchema>>;
-}>;
-
-function useForceUpdate() {
-  const [, setValue] = useState(0);
-
-  return () => setValue((value) => value + 1);
-}
+import { DependencyList } from "react";
+import { getDefaultReactSlashMenuItems } from "../SlashMenu/defaultReactSlashMenuItems";
 
 /**
  * Main hook for importing a BlockNote editor into a React project
  */
 export const useBlockNote = <BSchema extends BlockSchema = DefaultBlockSchema>(
-  options: Partial<
-    BlockNoteEditorOptions<BSchema> & {
-      customElements: CustomElements<BSchema>;
-    }
-  > = {},
-  deps: DependencyList = []
+  options: Partial<BlockNoteEditorOptions<BSchema>> = {},
+  _deps: DependencyList = []
```

Comment:
I think we still need to check this? Now we'll create a new instance whenever the hook is called

Reply:
I think a usememo should be enough